soccer/planning/PivotPathPlanner.cpp

```diff
@@ -1,38 +1,95 @@
 #include "PivotPathPlanner.hpp"
-#include "TrapezoidalPath.hpp"
 #include "EscapeObstaclesPathPlanner.hpp"
 #include <Configuration.hpp>
-#include <cmath>
-#include <boost/range/irange.hpp>
 #include "RRTPlanner.hpp"
 
 using namespace std;
 using namespace Geometry2d;
 
 namespace Planning {
 
-bool PivotPathPlanner::shouldReplan(MotionInstant startInstant,
-                                    const MotionCommand* cmd,
-                                    const MotionConstraints& motionConstraints,
-                                    const Geometry2d::ShapeSet& obstacles,
-                                    const Path* prevPath) {
-    PivotCommand command = *dynamic_cast<const PivotCommand*>(cmd);
-    debugThrow("Unfinished Class");
-    // TODO Implement This
-    return true;
+REGISTER_CONFIGURABLE(PivotPathPlanner);
+
+ConfigDouble* PivotPathPlanner::_pivotRadius;
+
+void PivotPathPlanner::createConfiguration(Configuration* cfg) {
+    _pivotRadius = new ConfigDouble(cfg, "Pivot/radius", 1.0);
```

Comment:
Nevermind, I see that it's a multiplier.  As of https://github.com/RoboJackets/robocup-software/issues/345, you can add a description string to Congig values - it might be nice to add one here